code/modules/mob/living/silicon/robot/login.dm

```diff
@@ -4,4 +4,3 @@
 	if(!. || !client)
 		return FALSE
 	regenerate_icons()
-	show_laws(0)
```

Comment:
Why was this removed?

Reply:
Sometimes `Login` is called at the same as `lawsync` causing `show_laws` to be called twice.

Reply:
On second thought it might be better to put in a check that returns if `show_laws` is called more than once per tick.

Reply:
After a second look this shouldn't be removed.